packages/app/index.html

```diff
@@ -4,7 +4,8 @@
   <meta charset="UTF-8">
   <link rel="icon" href="/favicon.ico" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Vite App</title>
+  <link rel="stylesheet" href="./index.css">
```

Comment:
The index.css should be imported via js code.

Reply:
fixed in 4bdb939bf1512fcb307e91af91ffd2fc6a71087f